### PR TITLE
almost there maybe??

### PR DESCRIPTION
**Pull request by shingoed**

### CODE REVIEW

**Comment by shingoed** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code401challenges/src/test/java/linkedlist/LinkedListTest.java [hunk 1]
```diff
@@ -118,6 +118,25 @@ public void testkthFromEnd() {
...
+        LinkedList c = new LinkedList();
+        a.insert(4);
+        a.insert(5);
+        a.insert(7);
+
+        b.insert(10);
+        b.insert(10);
+        b.insert(10);
+
+        assertEquals(4,10,5,10.7,10, LinkedList)
```
Comment (SharinaS) [inline code401challenges/src/test/java/linkedlist/LinkedListTest.java]: Remember that insert probably is inserting to the head, so think about the order of the list you expect. And, if I were you, I'd use my toString() method to test with. So, `c.toString()`, perhaps.
